deploy/003_deploy_optimism_spokepool.ts

```diff
@@ -22,13 +23,13 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   //    * Native USDC address on L2
   //    * CCTP token messenger address on L2
   const constructorArgs = [
-    "0x4200000000000000000000000000000000000006",
+    WETH[spokeChainId],
     3600,
     21600,
-    L2_ADDRESS_MAP[spokeChainId].l2Usdc,
+    USDC[spokeChainId],
     L2_ADDRESS_MAP[spokeChainId].cctpTokenMessenger,
   ];
-  await deployNewProxy("Optimism_SpokePool", constructorArgs, initArgs, spokeChainId === 10);
+  await deployNewProxy("Optimism_SpokePool", constructorArgs, initArgs, spokeChainId === CHAIN_IDs.OPTIMISM);
```

Comment:
Reminder to revert this change because it's no longer desirable to do this after https://github.com/across-protocol/contracts/pull/577

Reply:
Done.